tests/test_diagram.py

```diff
@@ -14,50 +14,58 @@ def railroad_debug(self) -> bool:
         """
         Returns True if we're in debug mode
         """
+        return True
```

Comment:
I assume this shouldn't have been committed?

Reply:
Gah, yes - I'll set this in my PyCharm config from now on.